primary/src/proposer.rs

```diff
@@ -227,6 +236,40 @@ impl<PublicKey: VerifyingKey> Proposer<PublicKey> {
 
                 // Advance to the next round.
                 self.round += 1;
+
+                // Implement a control system to ensure that some min amount of work
+                // happen in every round, and extend the rounds if it does not. Here we set a target
+                // number of batches committes per round `target_min_batch_per_round`. We then
+                // observe a weighted average of how many batches we commit per round and compute
+                // a multiplier of the max header delay to try to hit this number of rounds
+                // between 1 and `max_multiplier`.
+
+                // The number of rounds since we last committed.
+                let round_diff = self.round - self.metrics.core_metrics.last_commit_round();
```

Comment:
Given that we now support reconfiguration - which resets `round = 0` - let's take that into account , as in this case `self.round` can end up being lower than the `last_commit_round`.